compiler/src/iree/compiler/Dialect/HAL/Transforms/Passes.cpp

```diff
@@ -39,6 +40,36 @@ struct TransformOptions : public PassPipelineOptions<TransformOptions> {
       llvm::cl::desc("Whether to link hal.executable ops together."),
       llvm::cl::init(true),
   };
+  Option<PipelinePhase> start{
```

Comment:
I'm not super sure we want to expose this - since I don't think we need it here we should drop it or move it to a separate PR to talk about